Packs/Base/Scripts/ValidateContent/README.md

```diff
@@ -0,0 +1,57 @@
+Runs validation and linting on content items.
```

Comment:
This description is not sufficient. You can mention briefly what script accepts and should return. How it is used in XSOAR (that  the validation feature runs it) - in case customer will try to modify this file)

Reply:
done, please let me know what you think